rst2pdf/styles.py

```diff
@@ -363,7 +363,7 @@ def __init__(self, flist, font_path=None, style_path=None, def_dpi=300):
                         fname, pos = findfonts.guessFont(style[key])
 
                         if style[key] in embedded_fontnames:
-                            pass
+                            fontList = findfonts.autoEmbed(style[key])
                         else:
                             fontList = findfonts.autoEmbed(style[key])
```

Comment:
Seeing as these are now identical, we could move them out of the conditional are simply retain the rest of the else block.

```
fontList = findfonts.autoEmbed(style[key])
if style[key] not in embedded_fontnames:
    if fontlist:
        # ...
```